docs/source/conf.py

```diff
@@ -331,51 +343,51 @@ def run(self):
 ]
 
 # doctest config
-doctest_global_setup = """
-from collections import OrderedDict
+# doctest_global_setup = """
```

Comment:
I think we can just run the default setup within the `.. testcode::` directive. This will allow us to change at only one place in the future if any more setup comes in

Reply:
You are right, thanks for pointing out that ! Let's try to do as you suggest

Reply:
So shall I just delete the commented block completely?

Reply:
Yes, we can delete it